utils/casctl

```diff
@@ -3,17 +3,12 @@
 # Copyright(c) 2012-2021 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
 #
-
-import platform
 import sys
 
-min_ver = "3.6"
-ver = platform.python_version()
-if ver < min_ver:
+if sys.version_info < (3, 6):
```

Comment:
I would export the minimum version number to a separate variable (just as it was before), to have it in one place instead of multiple.
I don't know how much value it has in this case (since there are only two of them and they are pretty close to each other), but this condition could look like this:
`if sys.version_info < tuple(map(int, min_ver.split('.'))):`